package.json

```diff
@@ -32,7 +32,7 @@
     "zx": "./build/cli.js"
   },
   "engines": {
-    "node": ">= 16.0.0"
+    "node": ">= 14.0.0"
```

Comment:
Technically `@types/node` should be downgraded too, but there was some bugfix for one of the async hooks types that's not available in the node 14 version (could be easily worked around with a cast).

Reply:
Using cast to cover error is a bad decision.

Reply:
Bugfix was the wrong word, more like they took the opportunity to use a major node bump go change the the types to use tuples. I'm less concerned when there are tests.